src/components/login.js

```diff
@@ -70,24 +71,23 @@ function Login() {
                                 <div class="col-lg-6">
                                     <div class="card-body p-md-5 mx-md-4">
                                         <div class="text-center">
-                                            <img src="https://www.zohowebstatic.com/sites/zweb/images/commonroot/zoho-logo-web.svg"
-                                                style={{ width: "185px" }} alt="logo" />
+                                            <img src={logo}
+                                                className="loginLogo" alt="logo" />
                                             <br></br>
-                                            <h2 class="mt-1 mb-5 pb-1" style={{ color: "blueviolet" }}>Login</h2>
+                                            <h2 class="mt-1 mb-5 pb-1 loginTextColor">Login</h2>
                                         </div>
-
                                         <form method="post" onSubmit={loginData}>
                                             <div class="form-outline mb-4">
-                                                <label class="form-label">Email</label>
+                                                <label class="form-label">Username</label>
                                                 <input type="text" id="email" class="form-control"
                                                     placeholder="enter email address" name="email" onChange={handleChange} />
```

Comment:
If we are changing the Label from Email to User name then it is expected that we should do changes at all the required places... Like id, placeholder, name etc